test/support/exercise_test_case.ex

```diff
@@ -147,4 +151,21 @@ defmodule ElixirAnalyzer.ExerciseTestCase do
   def assert_comments(_, _, _) do
     :noop
   end
+
+  def find_exemplar_code(exercise_test_module) do
+    with {slug, _module} <-
+           Enum.find(@exercise_config, fn
+             {_slug, %{analyzer_module: ^exercise_test_module}} -> true
+             _ -> false
+           end),
+         {:ok, config_file} <-
+           [@concept_exercice_path, slug, @meta_config] |> Path.join() |> File.read(),
+         %{"files" => %{"exemplar" => [path]}} <- Jason.decode!(config_file),
+         exemplar_code <-
+           [@concept_exercice_path, slug, path] |> Path.join() |> File.read!() do
+      exemplar_code
```

Comment:
Is this readable? I really squeezed the power of `with` here...

Reply:
It is not 😁 I think the biggest problem are the multi-step clauses that don't fit on a single line. The mixture of `File.read` and `File.read!` is also confusing to me.  If this is for testing purposes only, can't everything just raise an error?

Reply:
Excellent point.

Reply:
Actually, I spoke too fast, this mustn't crash because it has to return `nil` for practice exercises.
I'll still rewrite it.

Reply:
I'm struggling with this. Every `!` has a precise reason for being there or not. I settled for adding comments for the moment.